releasenotes/notes/fix_9363-445db8fde1244e57.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed `#9363 <https://github.com/Qiskit/qiskit-terra/issues/9363>`__.
+    by labeling the non-registerless synthesis in the order that Tweedledum expects.
```

Comment:
This could maybe stand a shade more explanation.

Reply:
True. Adding more in 232e4e521ebb55e09a2ccab36bea5ff8869d0eb9